.env.development

```diff
@@ -1,3 +1,4 @@
 FIND_BASE_URL=https://api.publish-teacher-training-courses.service.gov.uk/api/v3/
 HOSTING_ENVIRONMENT_NAME=development
 STATE_CHANGE_SLACK_URL=
+PORT=3000
```

Comment:
This doesn't seem to make  a difference for me locally — possibly because `dotenv` loads after Puma?

Reply:
It would be good to straighten this out, though, because local emails under foreman sometimes point to `localhost:5100`, which is odd. I wonder if that's the port Sidekiq thinks it's running on.

Reply:
I noticed that port 5000 is also mentioned on the instructions of this issue: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3392 so other devs must be experiencing the same issue that I have. Could PORT already be configured to 3000 globally on your machine?

Reply:
I normally run the app with `rails s` pointing to `localhost:3000`
Only if I need redis (for things like generating test applications or setting up organisational permissions), I run it with foreman (`localhost:5000`)

Reply:
As the [Procfile](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/Procfile#L1) points to the [puma configuration](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/puma.rb#L13) the expectation is that foreman runs the app on port 3000.

Reply:
Looks like we're falling back to Foreman's default of `5000`. This statement explains why Sidekiq runs on 5100 I think https://github.com/ddollar/foreman/blob/c0b178c164396d6afe6fc3874abfd22f6cddbeae/lib/foreman/engine.rb#L271

Reply:
I think setting `PORT` in the `Procfile` fixes everything actually! Setting for Sidekiq so email URLs are consistent

<img width="789" alt="Screenshot 2020-11-12 at 16 35 21" src="https://user-images.githubusercontent.com/642279/98968038-17d79280-2505-11eb-8c99-e39add06e0de.png">

Reply:
☝️ worth checking that doesn't break Heroku

Reply:
How about loading dotenv at the top of the Procfile as it would mean all configuration is kept at one place and also enable changing the port in a single place (for anyone who may have something else running at port 3000).

```
require 'dotenv'
Dotenv.load
```

Reply:
This doesn't work — I don't think the `Procfile` is Ruby?

Reply:
You are right, it doesn't. 

There are a couple of other things that can be done to control when things are loaded, [look here](https://github.com/bkeepers/dotenv#note-on-load-order) but I think it's easy enough for anyone to find if they are having issues so I'll just remove this from .env.development and leave it in the example file for now.